frontend/src/components/worksheets/NewUpload/NewUpload.jsx

```diff
@@ -88,9 +96,7 @@ class NewUpload extends React.Component<{
             const createBundleData = getDefaultBundleMetadata(name || file.name, description);
             index += 1;
             let url = `/rest/bundles?worksheet=${ worksheetUUID }`;
-            if (after_sort_key) {
-                url += `&after_sort_key=${ after_sort_key }`;
-            }
+            url = this._setAfterSortKey(after_sort_key,url);

```

Comment:
I think it might be clearer if we just remove the `_setAfterSortKey` function, and just do this:

```
url += `&after_sort_key=${ after_sort_key || -1 }`;
```

Also, is there a chance `after_sort_key` could be equal to 0? If so, we should replace the above by `isNaN(after_sort_key) ? -1: after_sort_key`

Reply:
done